tastytoast/build.gradle

```diff
@@ -28,7 +28,7 @@ dependencies {
 apply plugin: 'com.github.dcendents.android-maven'
 apply plugin: 'com.jfrog.bintray'
 
-version = "0.0.4"
+version = "0.0.5"
```

Comment:
@anastr Version of the library is updated considering many things. revert this to previous one !